client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -224,6 +251,22 @@ const getAddressErrors = ( addressData, countriesData ) => {
 		}
 	}
 
+	if ( shouldValidatePhone ) {
+		// EasyPost requires an origin phone number for international shipments.
+		// This validation ensures that EasyPost will accept it, even though the phone may be invalid.
+		if ( ! phone ) {
+			errors.phone = translate( 'Origin phone number is required for international shipments' );
+		} else if (
+			10 !==
+			phone
+				.split( /\D+/g )
+				.join( '' )
+				.replace( /^1/, '' ).length
+		) {
+			errors.phone = translate( 'Invalid phone number' );
```

Comment:
Phone numbers can include more than 10 digits - for example when adding an extension. If a user tried to add a phone number like that, they may not realize what we want from them with the error message "Invalid phone number". Can we make it clear what we want and why?

"Customs forms require a 10-digit phone number. Please edit your phone number so it has at most 10 digits."

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/24974/commits/38e4dc095a548caa4328f797ea722c3dc7297a69